src/Misc/atoms/ActionsBar.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import styled from 'styled-components';
+import Button from '../../Form/atoms/Button';
+import ButtonWithIcon, { IButtonWithIcon } from '../../Form/atoms/ButtonWithIcon';
+
+const Container = styled.div``;
+const Title = styled.div`
+  font-family: ${({ theme }) => theme.fontFamily.ui};
+  color: hsl(207, 5%, 57%);
+  font-size: 12px;
+  font-weight: 700;
+  margin-bottom: 17px;
+`;
+
+const ButtonsWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const LeftButtons = styled.div`
+  button {
+    margin-right: 10px;
+    margin: 0 10px 10px 0;
+  }
+`;
+
+const RightButtons = styled.div`
+  flex-shrink: 0;
+  align-self: flex-end;
+`;
+
+const SelectedResults = styled.div`
+  font-family: ${({ theme }) => theme.fontFamily.ui};
+  margin-top: 40px;
+  min-height: 19px;
+  color: hsl(207, 5%, 57%);
+  font-size: 12px;
+`;
+
+
+const renderSelected = (template: string, selected: number, total: number) => {
+  const newSelected = template.replace('[SELECTED]', `${selected}`);
+  return newSelected.replace('[TOTAL]', `${total}`);
+};
+
+export interface IActionsButton extends IButtonWithIcon {
+  text: string
+}
+
+export interface IActionsBar {
+  title?: string
+  finishTextButton?: string
+  actionButtons?: IActionsButton[]
+  selectedTemplate?: string
+  totalSelected?: number
+  totalAvailable?: number
+  finishCallback?: () => void
+}
+
+const ActionsBar: React.FC<IActionsBar> = ({
+  title = 'Actions:',
+  finishTextButton = 'Finish',
+  actionButtons = [],
+  selectedTemplate = 'Selected [SELECTED] of [TOTAL] Results',
```

Comment:
I wonder if this could actually be a template literal instead of using our own injection points

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates

Reply:
Hi!

I might not understand correctly your question, I did read the link but couldn't understand what I'm not doing right. I think I'm doing template literals, although it might not be obvious

![Screen Shot 2022-07-21 at 19 43 20](https://user-images.githubusercontent.com/10409078/180195727-869c1cf2-b80c-42b6-9616-c27561ca6a57.png)

This selected value is the default template, the user can send any kind of sentence in any language while maintaining the [SELECTED] and [TOTAL] variables

The Developer can send something like

selectedTemplate = "Seleccionando [SELECTED] libros de [TOTAL]"

"Seleccionando 5 libros de 10"

The story has the example of English and Japanese


![Screen Shot 2022-07-21 at 19 37 02](https://user-images.githubusercontent.com/10409078/180195986-a4c78c67-d89a-4bea-99e9-449325c86fae.png)

![Screen Shot 2022-07-21 at 19 36 54](https://user-images.githubusercontent.com/10409078/180196021-d744c238-627c-40aa-85c5-f2508d8fc252.png)

Sorry  I couldn't see what I need to improve, please provide and example to confirm this part or let's chat on slack to review it.

Reply:
I understand WHY there are interpolation points, I am just suggesting not reinventing a feature of the language. the prop passed in could be the template sting instead of a string with custom made injection points.

Reply:
It seems harder than expected to use template literals as variables. 
This solution's seems fine.